src/blk/kernel/KernelDevice.cc

```diff
@@ -65,12 +65,10 @@ KernelDevice::KernelDevice(CephContext* cct, aio_callback_t cb, void *cbpriv, ai
     discard_callback(d_cb),
     discard_callback_priv(d_cbpriv),
     aio_stop(false),
-    discard_started(false),
-    discard_stop(false),
```

Comment:
One should include https://github.com/ceph/ceph/pull/59026 or leave discard_stop initialization to avoid valgrid errors